examples/ios/QuickStartSwiftUI/QuickStart.swift

```diff
@@ -22,6 +22,7 @@ let randomNouns = [
     "cork", "mouse pad"
 ]
 
+// MARK: Item
```

Comment:
Ah yes, that's the syntax I was looking for. How would you google this is you forgot "Mark"?

Reply:
I would google "#pragma mark swift equivalent."